### PR TITLE
Comment additions to conform to golint

### PR DESCRIPTION
**Pull request by orthogonous**
Also 1 slight code change that fixed an if statement
that golint didn't like.

### CODE REVIEW

File: services/precreator/service.go [hunk 1]
```diff
@@ -7,6 +7,7 @@ import (
 	"time"
 )
 
+// Service struct to manage the precreator sharding service.
```
Comment (otoolep) [inline services/precreator/service.go]: "Service manages the shard precreation service"

File: services/subscriber/service.go [hunk 2]
```diff
@@ -20,6 +20,8 @@ const (
 	statWriteFailures = "writeFailures"
 )
 
+// PointsWriter is an interface for writing points to db.
```
Comment (otoolep) [inline services/subscriber/service.go]: "to db." -> "to the database."

File: services/subscriber/udp.go [hunk 3]
```diff
@@ -6,15 +6,17 @@ import (
 	"github.com/influxdb/influxdb/cluster"
 )
 
-// Writes points over UDP using the line protocol
+// UDP writes points over UDP using the line protocol.
```
Comment (otoolep) [inline services/subscriber/udp.go]: "UDP writes" "UDP supports writing points..."

File: services/subscriber/udp.go [hunk 4]
```diff
@@ -6,15 +6,17 @@ import (
...
 type UDP struct {
 	addr string
 }
 
+// NewUDP returns a new UDP listener with default options.
 func NewUDP(addr string) *UDP {
 	return &UDP{addr: addr}
 }
 
+// WritePoints writes points over udp connection
```
Comment (otoolep) [inline services/subscriber/udp.go]: udp connection -> UDP transport

**Comment by otoolep** [comment]
Thanks @orthogonous -- can you tweak the comments as requested?

**Comment by orthogonous** [comment]
@otoolep thanks I will do that now and repush

**Comment by orthogonous** [comment]
@otoolep pushed and should be good now

**Comment by otoolep** [comment]
+1.

@nathanielc -- can you also review? It touches some of your code.

File: services/subscriber/service.go [hunk 5]
```diff
@@ -49,6 +51,7 @@ type Service struct {
 	mu              sync.Mutex
 }
 
+// NewService returns a subscriber service with default settings.
```
Comment (nathanielc) [inline services/subscriber/service.go]: "NewService returns a subscriber service with given settings"

File: services/subscriber/service.go [hunk 6]
```diff
@@ -20,6 +20,8 @@ const (
 	statWriteFailures = "writeFailures"
 )
 
+// PointsWriter is an interface for writing points to the database.
```
Comment (nathanielc) [inline services/subscriber/service.go]: "PointsWriter is an interface for writing points to a subscription destination."

**Comment by nathanielc** [comment]
Looks go to me once those two small comments are updated.

**Comment by orthogonous** [comment]
@otoolep updated as @nathanielc requested and re pushed

**Comment by otoolep** [comment]
Thanks @orthogonous

**Comment by orthogonous** [comment]
@otoolep just wondering if that tshirt promo is still going; would love to show the influx shirt at work

**Comment by otoolep** [comment]
Sure @orthogonous -- e-mail me your postal address.

philip at influxdb

**Pull request merged**